public/stylesheets/site/2.0/25-media-midsize.css

```diff
@@ -53,22 +53,34 @@
 
 #main, #main.dashboard {
   float: none;
-  margin: auto 1%;
+  margin: auto;
+  padding-left: 3.5%;
+  padding-right: 3.5%;
   width: auto;
-  padding-left: 0.5em;
-  padding-right: 0.5em;
 }
 
 /* 16 zone system */
 
-.splash {
```

Comment:
We should test how the padding looks for logged-out users when we come to testing, but this seems reasonable.